### DISCUSSION TITLE
WebUI models not being pulled/applied from site

### DISCUSSION THREAD

**Discussion by diffusionStable** [open-webui/open-webui#3712] [General]
# Bug Report

## Description
WebUI models not being pulled/applied from openweb-ui website

**Bug Summary:**
[Provide a brief but clear summary of the bug]
After having to reinstall my OS recently and re-installing open-webui,  when I try to pull the Based Dolphin Mixtral model it applies the modelfile but I have to then select a base model. This was not the case when I used this a few weeks ago, when I chose get, the model was downloaded too.

Even if I download the Dolphin Mixtral model from ollama and use it as the base model, I'm getting different performance now and the model is definitely less willing to answer in an uncensored manner. It also mentions its system prompt at the start of each new conversation, which was not the case when I downloaded the actual model from @hub. 

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]
1. Run open-webui
2. Navigate to workspace
3. Select "Discover a model"
4. When redirected to https://openwebui.com/
5. Pick any model from Top models
6. Select "Get"
7. Choose import to openwebui URL or download as JSON.
8. You are redirected back to localhost and forced to  choose a base model, which was previously not the case.

**Expected Behavior:**
[Describe what you expected to happen.]
When previously carrying out the above steps some weeks ago, after step 7, the model file was downloaded from the creator ie. "based-dolphin-mixtral:latest"
 
**Actual Behavior:**
[Describe what actually happened.]
I now have to  choose a base model myself.

## Environment

- **Open WebUI Version:** [3.7]
- **Ollama (if applicable):** [e.g., 0.1.30, 0.1.32-rc1]

- **Operating System:** [Ubuntu 22.04]
- **Browser (if applicable):** [ Firefox 127.0.2]

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by jsapede** [comment]
got same problem, is there any infos / solutions ?

**Comment by robertheadley** [comment]
I am also experiencing this. Installed through Pinokio.

**Comment by mccawley74** [comment]
Is this still an open? I am having the same issue, and have not found a work around.

**Comment by davidtavarez** [comment]
I am having the same issue.

**Comment by TacImpulse** [comment]
Ditto...

**Comment by Hoss83** [comment]
Same...

**Comment by St0nehead** [comment]
Same issue 🤬
